types/generated/contentTypes.d.ts

```diff
@@ -610,24 +610,22 @@ export interface ApiIapIap extends Struct.CollectionTypeSchema {
     draftAndPublish: true;
   };
   attributes: {
-    appleProductIDs: Schema.Attribute.Component<'iap.apple-product-i-ds', true>;
     createdAt: Schema.Attribute.DateTime;
     createdBy: Schema.Attribute.Relation<'oneToOne', 'admin::user'> &
       Schema.Attribute.Private;
-    googleSKUs: Schema.Attribute.Component<'iap.google-sk-us', true>;
     internalName: Schema.Attribute.String;
+    interval: Schema.Attribute.Enumeration<
+      ['monthly', 'sixmonthly', 'yearly']
```

Comment:
`sixmonthly` <-- should this be `halfyearly`? We use `halfyearly` elsewhere

Reply:
Hmm, that was changed/fixed for STG...